src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -102,3 +103,8 @@ def _check_orchestrator_host_op(self, orch_client, hostname, add_host=True):
                 msg='Remove a non-existent host {} from orchestrator'.format(
                     hostname),
                 component='orchestrator')
+
+    @RESTController.Resource('GET')
+    def devices(self, hostname):
+        # (str) -> dict
+        return CephService.send_command('mon', 'device ls-by-host', host=hostname)
```

Comment:
Since we are returning the response of the mon command directly, we should have a backend QA test validating the structure of the response to catch any change on the `device ls-by-host` command output.